CONTRIBUTING.adoc

```diff
@@ -0,0 +1,184 @@
+* *Bug report*: If you’re reporting an error in the existing documentation, use this template. This could be anything from broken samples to typos. When you create a bug report, include as many details as possible and include suggested fixes to the issue. If you know which Knative component your bug is related to, you can assign the appropriate https://github.com/knative/community/blob/master/working-groups/WORKING-GROUPS.md[Working Group Lead].
+* *Feature request*: For upcoming changes to the documentation or requests for more information on a particular subject.
+
+Note that code issues should be filed against the http://github.com/knative[individual Knative repositories], while documentation issues should go in the https://github.com/knative/docs/issues[`knative/docs` repo]. If the issue is specific to the https://knative.dev website, open the issue in the https://github.com/knative/website/issues[`knative/website` repo].
+
+
+=== Fixing Issues (Pull Requests)
+
+////
+This could use a pass to be more focused on what a PR submitter should do at the start of the process.
+////
+
+Here’s what generally happens when you open a PR against the `knative/docs` repo:
+
+. One of the assigned repo maintainers will triage the PR by assigning relative priority, adding appropriate labels, and performing an initial documentation review.
+. If the PR involves significant technical changes, for example new features, or new and changed sample code, the PR is assigned to a Subject Matter Expert (SME), typically an engineer working on Knative, for technical review and approval.
+. When both the technical writers and SMEs are satisfied with the quality of the writing and the technical accuracy of the content, the PR can be merged. A PR requires two labels before it can merge: `lgtm` and `approved`.
+* The SME is responsible for reviewing the technical accuracy and adding the `lgtm` label.
+* The https://github.com/knative/docs/blob/master/OWNERS_ALIASES[Knative technical writers] are who provide the `approved` label when the content meets quality, clarity, and organization standards (see link:#style-guide[Style Guide]).
+
+We appreciate contributions to the docs, so if you open a PR we will help you get it merged. You can also post in the `#docs` https://knative.slack.com/[Slack channel] to get input on your ideas or find areas to contribute before creating a PR.
+
+== Getting involved
+
+There are a couple of different ways to get started with contributing to the Knative documentation:
+
+* Look for a https://github.com/knative/docs/labels/kind%2Fgood-first-issue[Good First Issue] in the backlog.
+
+* Try out Knative and send us feedback. For example, run through the link:./docs/install/[install guides] and then try link:./docs/serving/getting-started-knative-app.md[Getting Started with Knative Serving] or link:./docs/eventing/getting-started.md[Getting Started With Eventing].
```

Comment:
How do they send us feedback? Maybe send them to the #user-experience channel?? *shameless self plug*

Reply:
How about filing an issue with the friction log attached? With that said, I defer to your preferences.

Reply:
That feels like a big lift for a new user, until it becomes unwieldy, I am ok with folks coming to UX channel and yelling at us!